src/components/settings/Users/AddUser.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+    <h4 class="mt-4">
+        {{ $t("Create an admin account") }}
+    </h4>
+
+    <form data-cy="setup-form" @submit.prevent="create">
```

Comment:
I'm doing it this way especially because it's "as with the other forms" of the webapp:
![image](https://github.com/M1CK431/uptime-kuma/assets/31622263/bf02d363-50be-4bc1-a4a4-87268b08c4ff)

Reply:
`<form submit="...">` handles pressing Enter key as well as pressing the Button with `type=submit`.